app/api/eventListeners.ts

```diff
@@ -1,8 +1,10 @@
 import { EventsBus } from './eventsbus';
+import { registerEventListeners as registerSegmentationListeners } from './services/pdfsegmentation/eventListeners';
 import { Suggestions } from './suggestions/suggestions';
 
 const registerEventListeners = (eventsBus: EventsBus) => {
   Suggestions.registerEventListeners(eventsBus);
```

Comment:
As a note, I think we should take the register function out of the Suggestions object and leave it standalone just as the one for segmentation.